src/equations/equations.jl

```diff
@@ -239,11 +239,13 @@ The return value will be `True()` or `False()` to allow dispatching on the retur
 """
 have_nonconservative_terms(::AbstractEquations) = False()
 """
-    nnoncons(equations)
-Number of nonconservative terms for a particular equation. The default is 0 and 
-it must be defined for each nonconservative equation independently.
+    n_nonconservative_terms(equations)
+
+Number of nonconservative terms in the form local * symmetric for a particular equation.
+This function needs to be specialized only if equations with nonconservative terms are
+combined with certain solvers (e.g., subcell limiting).
 """
-nnoncons(::AbstractEquations) = 0
+function n_nonconservative_terms(::AbstractEquations) end
```

Comment:
What you're implementing here is a method with an empty body, defaulting to a return value of `nothing`. What you want is a function declaration _without_ a method, i.e.,
```suggestion
function n_nonconservative_terms end
```